src/GeneratorV2/Process/Processor.cs

```diff
@@ -16,6 +16,7 @@ public static class Processor
         // FIXME: Maybe we dont want classes for these?
         static readonly IOverloader[] Overloaders = new IOverloader[]
         {
+            new VectorOverloader(),
```

Comment:
Why is the vector overloader before everything?

Reply:
Because it has to be before the trimname overloader, as we are partly looking at the name as part of the vectoroverloader. However this might be able to move anywhere above the span array overloader if we fix the trimnameoverloader entirely.